test/integration/chat-workflow.test.js

```
@@ -75,14 +86,31 @@ QUnit.module('Chat Workflow', {
         this.mockKanbn = new MockKanbn();
 
         mockRequire('../../src/main', {
-            Kanbn: () => this.mockKanbn,
+            Kanbn: function() { return this.mockKanbn; }.bind(this),
             findTaskColumn: (index, taskId) => {
-                for (const [column, tasks] of Object.entries(index.columns)) {
-                    if (tasks.includes(taskId)) {
-                        return column;
+                if (index && index.columns) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)